classes.py

```diff
@@ -0,0 +1,14 @@
+class User:
+    def __init__(self, name: str, email: str, password: str):
+        self.name = name
+        self.email = email
+        self.password = password
```

Comment:
Consider using hashing and then storing the passwords.

Reply:
Yes, you will be able to see all the functionalities and better complexity of the code in the upcoming PRs.